### ISSUE TITLE
fcn8s on Pascal VOC

### ISSUE THREAD

**Issue by hihfs** [meetshah1995/pytorch-semseg#120]
Hi,

I am trying to reproduce the results in the [PAMI version of the FCN paper](https://arxiv.org/pdf/1605.06211.pdf), training fcn8s on Pascal VOC with the default setting. In the paper (Table 4), the authors report the mean IOU of 67.2 on VOC2012. I can't seem to achieve anything near it on the validation set during the training (currently it is ~62). What is the difference, do you suppose?

Also, I am wondering if the argument 'split' in the code:
```
    v_loader = data_loader(
        data_path,
        is_transform=True,
        split="val",
        img_size=(args.img_rows, args.img_cols),
        img_norm=args.img_norm,
    )
```
in train.py should be "train_aug_val" instead of "val". As it is, the validation set has an intersection with the training set "train_aug". If we use it, wouldn't the IOU become even lower?
Also, which set in your repo is the "test set"  authors of the above paper refer to?

**Comment by Feiyu-Zhang** [comment]
Mean IOU is 65.7 on the validation set. But I don's test on the test set.
